app/src/main/java/org/mozilla/tv/firefox/utils/ServiceLocator.kt

```diff
@@ -44,13 +44,13 @@ import org.mozilla.tv.firefox.pocket.PocketVideoRepo
  *   open val telemetry: TelemetryInterface by lazy { SentryWrapper() }
  *   ```
  */
-open class ServiceLocator(val app: Application) {
+open class ServiceLocator(val app: Application, getCurrentLanguage: () -> String) {
```

Comment:
See above about having redundant dependencies. There could be a better resolution while satisfying mockability in testing